vault/resource_jwt_auth_backend_role.go

```diff
@@ -98,6 +106,24 @@ func jwtAuthBackendRoleResource() *schema.Resource {
 					Type: schema.TypeString,
 				},
 			},
+			"oidc_scopes": {
+				Type:        schema.TypeSet,
+				Optional:    true,
+				Description: "List of OIDC scopes to be used with an OIDC role. The standard scope \"openid\" is automatically included and need not be specified",
+				Elem: &schema.Schema{
+					Type: schema.TypeString,
+				},
+			},
+			"bound_claims": {
```

Comment:
wondering if that should be plural or singular

Reply:
Using plurals to be in line with Vault's documentation - https://www.vaultproject.io/api/auth/jwt/index.html